docs(ins): review content (#3330)
scaleway/docs-content: containers/kubernetes/how-to/enable-easy-deploy.mdx

1. Click **Kubernetes** in the **Containers** section of the [Scaleway console](https://console.scaleway.com) side menu. The Kubernetes Kapsule overview displays.
2. Click <Icon name="edit" /> next to the container you wish to edit, then click **Edit** in the menu. A pop-up displays with the configuration options.
3. When modifying an application deployed from **Container Registry**, modify the container parameters according to your requirements, then click **Update container** to conclude.
    <Lightbox src="scaleway-update-container.webp" alt="" />
    <Message type="note">

>       The container **type** cannot be modified. If you wish to use a different type, a new container must be created.

Reviewer: ```suggestion
      It is not possible to modify a container **type**. If you wish to use a different type, you must create a new container.
```
